x-pack/plugins/siem/public/components/timeline/data_providers/providers.tsx

```diff
@@ -39,6 +39,13 @@ interface Props {
   onToggleDataProviderExcluded: OnToggleDataProviderExcluded;
 }
 
+/**
+ * This fixed height prevents the timeline's droppable area from growing,
+ * (growth causes layout thrashing) when the AND drop target in a row
+ * of data providers is revealed.
+ */
+const ROW_OF_DATA_PROVIDERS_HEIGHT = 43; // px
+
```

Comment:
General note and observation (no actionable item asked): Once we move PanelProviders below to `EuiGroup` and `EuiFlexItem` we will have a decent shot at cross browser compatibility with IE-11.

Reply:
i'd prefer to sweep through the IE11 changes in another PR

Reply:
Sounds good 👍